utils/audioLoop.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
I'm going to be pedantic and request that filenames are hyphenated (so `audio-loop.js`).